scripts/activate-protocol.sh

```diff
@@ -23,12 +23,13 @@ usage() {
     echo "  --tezos-client <filepath>. Path for patched tezos-client executable"
     echo "  --parameters <filepath>. Path to JSON file with protocol parameters."
     echo "  [--fitness <int>]. Protocol activation fitness, default value is $fitness."
+    echo "  [--base-chain <babylonnet | carthagenet>]. Define base chain for your private"
+    echo "    blockchain. Default is 'babylonnet'."
     echo "  [--protocol <protocol-name>]. Protocol to activate, default value is"
-    echo "  $protocol"
+    echo "  'PsBabyM1eUXZseaJdmXFApDSBqj8YBfwELoxZHHW77EMcAbbwAS'."
 
 }
 
-protocol="PsBabyM1eUXZseaJdmXFApDSBqj8YBfwELoxZHHW77EMcAbbwAS"
```

Comment:
Why? I mean, why not define `protocol_babylonnet` or something like that? This constant is now present in two places and it does not look nice.

Reply:
@rvem ping

Reply:
:man_facepalming: